app/models/manageiq/providers/container_manager.rb

```diff
@@ -30,5 +32,10 @@ def aggregate_cpu_total_cores(targets = nil)
     def aggregate_memory(targets = nil)
       aggregate_hardware(:computer_systems, :memory_mb, targets)
     end
+
+    def add_default_policies
+      # make sure that add_policy indeed adds a policy profile (aka MiqPolicySet)
+      add_policy(MiqPolicySet.find_by_guid('747a70d0-a8bf-4b63-bdad-1676bc0c5356'))
```

Comment:
@moolitayer Regarding the guids - we added them to allow for creating policies (and other things that may not be policies) in one database, exporting them and importing them in another DB. And, updating them if they already exist in the destination DB. This was necessary because you can't rely on the ids being the same for the same policy in both DBs. So, the guid ensures that we can match them.

This was a common pattern that was used in the field so that policies, some that were very complex, can be built and shared.

> Sure. Is it alright to seed a tag connected to that policy and then connect all tagged policies? Are tags for policies supported? (I'm trying to avoid the db change)

Which db change do you mean? Changing the name value?